content/en/docs/_index.md

```diff
@@ -1,9 +1,9 @@
 ---
 title: Operator Lifecyle Manager Documentation
-linkTitle: OLM Documentation
+linkTitle: "OLM"
```

Comment:
FYI: sneaking in this change here

Reply:
I like this, but we should try to be consistent with SDK's site (at least for summit): https://sdk.operatorframework.io/

It's just `Documentation` there.